README.md

```diff
@@ -44,6 +44,7 @@ and how it advertises itself to a Cryostat server instance. Required properties
 - [ ] `cryostat.agent.app.name` [`String`]: a human-friendly name for this application. Default `cryostat-agent`.
 - [ ] `cryostat.agent.app.jmx.port` [`int`]: the JMX RMI port that the application is listening on. The default is to attempt to determine this from the `com.sun.management.jmxremote.port` system property.
 - [ ] `cryostat.agent.registration.retry-ms` [`long`]: the duration in milliseconds between attempts to register with the Cryostat server. Default `5000`.
+- [ ] `cryostat.agent.exit.deregistration.timeout` [`Duration`]: the duration to wait for a response from the Cryostat server when attempting to deregister at shutdown time . Default `3s`.
```

Comment:
Should this be specific in that it needs a property of ms?

Reply:
Not sure what happened here, I already made that change in one of the last commits that was pushed before this comment, but it isn't showing this comment as outdated...

Reply:
I was trying to get SmallRye Config to let me use Duration instead of `long` so that the config values could be like `1s` or `500ms` instead of always being written out in `ms`, but it needs some extra converter added to parse text to Durations like that, so I dropped it and went back to just specifying it in `ms`.